scripts/collect_api_events.rb

```diff
@@ -38,38 +44,55 @@ def parse_body(response_body)
   body
```

Comment:
I'm not sure this `body` variable is needed? I don't see any circumstance where the output of this function would the unmodified empty hash that's originally assigned to the variable. The variable is assigned an empty hash and then an attempt is made to assign it again with whatever `JSON.parse` returns. If that's an error we don't use `body`. If it's not an error we get the `JSON.parse` result. Either way, we don't use the empty hash. I think this could be shortened to just the `begin..rescue..end` block, and you don't have to bother assigning `JSON.parse` to a variable. Just let the implicit return take care of it.